src/table/components/TablePaginationActions.jsx

```diff
@@ -51,31 +55,42 @@ export default function TablePaginationActions({
         aria-label={translator.get(`SNTable.Pagination.${type}`)}
         title={!constraints.passive && translator.get(`SNTable.Pagination.${type}`)}
         tabIndex={tabIndex}
-        sx={disabledCondition ? { color: 'rgba(0, 0, 0, 0.3)', cursor: 'default' } : { color: 'rgba(0, 0, 0, 0.54)' }}
+        sx={disabledCondition ? { color: disabledIconButtonColor, cursor: 'default' } : { color: iconButtonColor }}
         onKeyDown={onKeyDown}
       >
         <IconComponent />
       </IconButton>
     );
   };
 
+  const inputProps = {
+    'data-testid': 'pagination-dropdown',
+    tabIndex,
+    id: 'pagination-dropdown',
+    style: {
+      color: theme.isBackgroundDarkColor ? 'rgba(255, 255, 255, 0.9)' : '#404040',
```

Comment:
You are doing this check three times. create a variable and do it once

Reply:
Good point, I gonna do the refactor in another PR as I found more cases like this one.

Reply:
#510